tests/publisher/snaps/tests_post_release.py

```diff
@@ -0,0 +1,100 @@
+import responses
+
+from tests.publisher.endpoint_testing import BaseTestCases
+
+
+class PostReleasePageNotAuth(BaseTestCases.EndpointLoggedOut):
+    def setUp(self):
+        snap_name = "test-snap"
+        endpoint_url = "/account/snaps/{}/release".format(snap_name)
+
+        super().setUp(
+            snap_name=snap_name,
+            endpoint_url=endpoint_url,
+            method_endpoint="POST",
+        )
+
+
+class PostDataReleasePage(BaseTestCases.EndpointLoggedIn):
```

Comment:
Would be good to have a test than errors return as JSON?

Reply:
done